### ISSUE TITLE
Tagalog translation: new booster content as of 1/11/23

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#51919]
## Issue Description

We made a few updates to 2 COVID-19 benefit detail pages that we need translated in Tagalog.

On the [main vaccines page](https://www.va.gov/health-care/covid-19-vaccine/) we made updates to the blue eligibility box.

On the [boosters](https://www.va.gov/health-care/covid-19-vaccine/booster-shots-and-additional-doses/) page we made the same changes in the "When to get your updated booster shot section."  This wasn't sent explicitly to the translators since it's the same same sentences on the first page. 

We need to update the Tagalog translation pages with these edits. 

Word doc sent to translators: [https://app.zenhub.com/files/133843125/48350f63-eb47-4618-9f61-b1ff1b74799a/download](https://app.zenhub.com/files/133843125/48350f63-eb47-4618-9f61-b1ff1b74799a/download)

## Tasks
- [x] Update main vaccine page (Aliyah)
- [x] Update booster page (Aliyah)
- [x] Copyedit main vaccine page (Megan)
- [x] Copyedit booster page (Megan)
- [x] Publish main vaccine page (Megan)
- [x] Publish booster page (Megan)

## Acceptance Criteria
- [x] Tagalog pages are live with updates

**Comment by aliyahblackmore** [comment]
Thanks Randi! 

@megzehn this is ready for you! 

Main COVID-19 page 
**Preview link**: http://preview-prod.vfs.va.gov/preview?nodeId=16413
Notes: I noticed that there was a different word used for "Note" on the current live translated page. I changed this to "Paalala" to be consistent with what we've used across Tagalog pages.

Boosters page
Preview link: http://preview-prod.vfs.va.gov/preview?nodeId=34394
**Notes:**

1. I removed this text "Magpabakuna ng makabagong booster na inikyon na hindi bababa sa 2 buwan matapos makumpleto ang unang serye ng bakuna o ang huling booster" as it is no longer on the live English page. 

2. I updated the note in this section to match what seems to be the most up to date information, from the main COVID page.
I believe we need to update this on the live English page @RLHecht
